spec/factories/green_flag/feature_event.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :green_flag_feature_event, class: GreenFlag::FeatureEvent do
+    event_type_code 'test_event'
+    count           1
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.